compiler/src/iree/compiler/Dialect/LinalgExt/IR/AggregatedOpInterfaceImpl.cpp

```diff
@@ -339,10 +361,13 @@ OnlineAttentionOp::decomposeOperation(OpBuilder &b) {
   Value emptyS = b.create<tensor::EmptyOp>(loc, sSizes, elementType);
   Value sZero = b.create<arith::ConstantOp>(loc, b.getZeroAttr(elementType));
   Value s = b.create<linalg::FillOp>(loc, sZero, emptyS).getResult(0);
+
   s = computeMatmul(b, loc, getQueryMap(), getKeyMap(), sMap, query, key, s);
+  s.getDefiningOp()->setAttr("attention_qk_matmul", b.getUnitAttr());
+
+  s = applyRegion(b, loc, getRegion(), s);
   // TODO: We shouldn't be relying on such attributes. We need a better
   // mechanism to identify attention matmuls.
```

Comment:
the comment should be moved before the setAttr

Reply:
Fixed.